src/wazuh_db/wdb.c

```diff
@@ -147,6 +147,59 @@ int wdb_open_global() {
     return 0;
 }
 
+// Opens global database and stores it in DB pool. It returns a locked database or NULL
+wdb_t * wdb_open_global2() {
+    char path[PATH_MAX + 1];
```

Comment:
Please, initialize the array to zero. When the arrays are local arrays (basically a piece of memory in the stack) they are created by decreasing the SP but without cleaning the memory. So, is a good practice to initialize them either with `= { 0 }` as well as `= ""` (for the case of `char` arrays).